src/accessories/platformAccessory.ts

```diff
@@ -13,7 +14,98 @@ export abstract class MiHomePlatformAccessory {
     protected readonly platform: MiHomeGatewayPlatform,
     protected readonly accessory: PlatformAccessory,
   ) {
-    // TODO
+
+    // set accessory information
+    this.accessory.getService(this.platform.Service.AccessoryInformation)!
+      .setCharacteristic(this.platform.Characteristic.Manufacturer, MANUFACTURER)
+      .setCharacteristic(this.platform.Characteristic.Model, this.accessory.context.device_type)
+      .setCharacteristic(this.platform.Characteristic.Name, this.accessory.context.label)
+      .setCharacteristic(this.platform.Characteristic.AppMatchingIdentifier, APP_MATCHING_IDENTIFIER);
+  }
+
+  /**
+   * Handle the "GET" requests from HomeKit
+   * These are sent when HomeKit wants to know the current state of the accessory.
+   */
+  async getName(callback: CharacteristicGetCallback) {
```

Comment:
Are these method names a Homebridge convention thing? i.e. If we declare a characteristic called "MyThing" then we have methods `getMyThing` and `setMyThing`?

Reply:
The closest thing to a `convention` seems to be the Homebridge plugin template which is updated more frequently than anything else.